keyboards/archerkeyboard/desire65/keyboard.json

```
@@ -0,0 +1,125 @@
+{
+    "manufacturer": "archerkeyboard",
+    "keyboard_name": "desire65",
+    "maintainer": "akyp",
+    "backlight": {
+        "driver": "software",
+        "levels": 8,
+        "pin": "GP18"
+    },
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP21", "GP29", "GP1", "GP0", "GP15", "GP14", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP2", "GP5", "GP4", "GP3"],
+        "rows": ["GP24", "GP25", "GP19", "GP16", "GP17"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "default": {
+            "val": 128
+        },
+        "driver": "ws2812",
+        "led_count": 17,
+        "sleep": true
+    },
+    "url": "https://archerkeyboard.com",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0002",
+        "vid": "0x0361"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP26"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Add Community Layout support part 1
```suggestion
    "community_layouts": ["65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"],
    "layouts": {
        "LAYOUT_65_ansi_blocker_tsangan_split_bs": {
```